### PR TITLE
Re #1725 Add horizontal axis

### LINKED ISSUE #1725: Horizontal line at y=0 needed in P(r) plots
It would be helpful to have a horizontal line at y=0 on the P(r) plots in order to allow users a quick visual to determine whether or not P(r) goes to zero at the chosen Dmax or at values below the chosen Dmax.
lyso1_pofr

### PR DESCRIPTION
**Pull request by mdoucet**
Fixes #1725.

To test, use the P(R) calculator with test data and see the horizontal line at y=0 on the P(R) plot.

### CODE REVIEW

File: src/sas/qtgui/Plotting/Plotter.py [hunk 1]
```diff
@@ -222,6 +222,10 @@ def plot(self, data=None, color=None, marker=None, hide_error=False, transform=T
                             zorder=1,
                             picker=True)
 
+        # Display horizontal axis if requested
+        if data.show_yzero:
+            ax.axhline(color='black', linewidth=1)
```
Comment (rozyczko) [inline src/sas/qtgui/Plotting/Plotter.py:227]: I find `linewidth=1` slightly thick - the same thickness as axes. Maybe 0.5 would look nicer? or not...

**Review by rozyczko — APPROVED** [review]
Code looks good, thanks for cleanup in Plotter.py
Ready to merge.

**Comment by butlerpd** [comment]
ready for testing on Win

File: src/sas/qtgui/Plotting/Plotter.py (hunk 1 above)
Comment (mdoucet) [reply src/sas/qtgui/Plotting/Plotter.py:227]: I wonder if there's a way to automatically set it to the thickness of the borders.

Comment (krzywon) [reply src/sas/qtgui/Plotting/Plotter.py:227]: > I wonder if there's a way to automatically set it to the thickness of the borders.

Maybe something like `ax.axhline(color='black', linewidth=self.ax.spines['left'].get_linewidth())`?

**Review by butlerpd — APPROVED** [review]
Works as intended. Code looks fine to my untrained eye.  My lack of esthetics means i'm not qualified to judge the line thickness question 😄 which looks fine to me.

Ready to merge as far as I'm concerned

**Review by krzywon — APPROVED** [review]
Looks good. Feel free to merge whether or not you make the line width change.

**Pull request merged**